rust/windows-client/src-tauri/src/client/gui.rs

```diff
@@ -275,6 +281,9 @@ struct Controller {
     ctlr_tx: CtlrTx,
     /// connlib / tunnel session
     connlib_session: Option<connlib_client_shared::Session<CallbackHandler>>,
+    /// Hexadecimal SHA256 of the on-disk UUIDv4
+    /// Sent verbatim to Session::connect
+    hashed_device_id: String,
```

Comment:
let's use the `Uuid` type, it implements serialize/deserialize

Reply:
Ah yes, @ReactorScram I think Session automatically handles hashing to sha256 [here](https://github.com/firezone/firezone/blob/786e1563371aafe3ab4d3e10efb05b3fb4b4de5f/rust/connlib/shared/src/lib.rs#L47) so hashing in the client is unnecessary. Sorry for the confusion (getting hard to stay on top of things)

Reply:
Great, I'll drop the SHA256 and make it Uuid here.
I'll add a comment in the connlib docs too, should just be a couple lines

Reply:
Dropped the SHA256 in 4b351d7c
It does use Uuid internally, but I would prefer to keep this as a String so that it's passed verbatim to connlib without calling `to_string` anywhere except inside `device_id.rs`.